redis/ocsp.py

```diff
@@ -11,8 +11,50 @@
 from redis.exceptions import AuthorizationError, ConnectionError
 
 
+def _check_certificate(certificate):
+    """A wrapper the return the validity of a known ocsp certificate"""
+
+    ocsp_response = ocsp.load_der_ocsp_response(certificate)
+    if ocsp_response.response_status == ocsp.OCSPResponseStatus.UNAUTHORIZED:
+        raise AuthorizationError("you are not authorized to view this ocsp certificate")
+    if ocsp_response.response_status == ocsp.OCSPResponseStatus.SUCCESSFUL:
+        if ocsp_response.certificate_status == ocsp.OCSPCertStatus.REVOKED:
+            return False
```

Comment:
What if the Cert status is `UNKNOWN`? it is a possible status and I think you should return `False` on that as well

Reply:
It did in a previous version, restoring. I now check the OCSPResponseStatus and return there - but the next push restores this - excatly here. Good call!